charts/gardener/controlplane/values.yaml

```diff
@@ -2,7 +2,10 @@ global:
   # Gardener API server configuration values
   apiserver:
     enabled: true
-    serviceEnabled: true
+    service:
+      enabled: true
```

Comment:
That's a breaking change, perhaps you should mention it via release note.

Reply:
Thanks for the reminder! I had it in mind yesterday when I was doing the change but forgot the write down the release note. I now added a release note.